src/components/AppSidebarTab/AppSidebarTab.vue

```diff
@@ -67,7 +69,8 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-section {
+.app-sidebar__tab {
+	display: none;
```

Comment:
By moving this to the css, we allow to use v-show on the AppSidebarTab as well, otherwise it was conflicting as we'd have two handlers, from the inside and from the outside.